Until https://reviews.freebsd.org/D29734 lands revert
https://gitlab.freedesktop.org/mesa/mesa/-/commit/2c1e4c4baa81
https://gitlab.freedesktop.org/mesa/mesa/-/commit/75a725e4c555

meson.build:1480:4: ERROR: Invalid version of dependency, need 'libdrm_amdgpu' ['>=2.4.105'] found '2.4.104'.

../src/amd/common/ac_gpu_info.c:528:11: error: implicit declaration of function 'amdgpu_query_video_caps_info' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
      r = amdgpu_query_video_caps_info(dev, AMDGPU_INFO_VIDEO_CAPS_DECODE,
          ^
../src/amd/common/ac_gpu_info.c:528:11: note: did you mean 'amdgpu_query_heap_info'?
/usr/local/include/libdrm/amdgpu.h:1141:5: note: 'amdgpu_query_heap_info' declared here
int amdgpu_query_heap_info(amdgpu_device_handle dev, uint32_t heap,
    ^

--- meson.build.orig	2021-04-14 06:35:30 UTC
+++ meson.build
@@ -1452,7 +1452,7 @@ dep_libdrm_radeon = null_dep
 dep_libdrm_nouveau = null_dep
 dep_libdrm_intel = null_dep
 
-_drm_amdgpu_ver = '2.4.105'
+_drm_amdgpu_ver = '2.4.100'
 _drm_radeon_ver = '2.4.71'
 _drm_nouveau_ver = '2.4.102'
 _drm_intel_ver = '2.4.75'
--- src/amd/common/ac_gpu_info.c.orig	2021-04-14 06:35:30 UTC
+++ src/amd/common/ac_gpu_info.c
@@ -524,22 +524,6 @@ bool ac_query_gpu_info(int fd, void *dev_p, struct rad
    info->gart_size_kb = DIV_ROUND_UP(info->gart_size, 1024);
    info->vram_size_kb = DIV_ROUND_UP(info->vram_size, 1024);
 
-   if (info->drm_minor >= 41) {
-      r = amdgpu_query_video_caps_info(dev, AMDGPU_INFO_VIDEO_CAPS_DECODE,
-            sizeof(info->dec_caps), &(info->dec_caps));
-      if (r) {
-         fprintf(stderr, "amdgpu: amdgpu_query_video_caps_info for decode failed.\n");
-         return r;
-      }
-
-      r = amdgpu_query_video_caps_info(dev, AMDGPU_INFO_VIDEO_CAPS_ENCODE,
-            sizeof(info->enc_caps), &(info->enc_caps));
-      if (r) {
-         fprintf(stderr, "amdgpu: amdgpu_query_video_caps_info for encode failed.\n");
-         return r;
-      }
-   }
-
    /* Add some margin of error, though this shouldn't be needed in theory. */
    info->all_vram_visible = info->vram_size * 0.9 < info->vram_vis_size;
 
